clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -165,7 +165,8 @@ public String toString() {
             return "(error=" + error + ", highWaterMark=" + highWatermark +
                     ", lastStableOffset = " + lastStableOffset +
                     ", logStartOffset = " + logStartOffset +
-                    ", abortedTransactions = " + abortedTransactions + ", records=" + records + ")";
+                    ", abortedTransactions = " + abortedTransactions +
+                    ", recordsSizeInBytes=" + records.sizeInBytes() + ")";
```

Comment:
Since you're doing this, maybe you want to take the chance to implement `toString(boolean)` in `Fetcher` to make request logging more concise in `debug` versus `trace`? We did that for the `ProduceRequest` previously.